library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -134,6 +134,14 @@ public int getFrameCount() {
     return state.frameLoader.getFrameCount();
   }
 
+  /**
+   * Returns the current frame index in the range of 0 to {@link #getFrameCount()}-1.
```

Comment:
nit: remove 'range of 0 to {@link #getFrameCount()} -1 or -1 if no frame is displayed.

Remove '.' and update the 'of's in the second line to 'or' and 'if' respectively